plugins/dstat_nvidia_gpu.py

```diff
@@ -0,0 +1,41 @@
+### Author: Vasilis Vryniotis <bbriniotis@datumbox.com>
+
+class dstat_plugin(dstat):
+    """
+    Total GPU usage for NVIDIA cards. Requires the nvidia-ml-py package.
+
+    Usage: dstat --nvidia-gpu
+    """
+
+    def __init__(self):
+        self.name = 'gpu nvidia'
+        self.nick = ('gpu',)
+        self.vars = ('gpu',)
+        self.type = 'p'
+        self.width = 5
+        self.scale = 1
+
+    def check(self):
+        import pynvml
```

Comment:
If the import fails, we have to catch it and provide a proper error to the user to assist them.
```python
        try:
            import pynvml
        except:
            raise Exception, 'Needs python-XXXX module'
```